### ISSUE TITLE
Exceptions to the Autoteleport JIP func

### ISSUE THREAD

**Issue by KillPact** [SignalCommunity/SCMF#48]
Add exceptions to the autoteleport jip function so that pilots do not automatically get teleported to leadership elements.

**Comment by Winifridal** [comment]
I would assume this also would include vehicle crews (tanks, IFVs etc) too?

**Comment by KillPact** [comment]
No, just because its fine if those guys get teleported and often PLTLDR will be just a walk away from a tank possibly. The chances of needing to re-teleport those guys is less if they get moved to leadership than left at base whereas they will have to be teleported 100% of the time. Its at the point of trying to minimize our amount of work as possible. Also, if PLTLDR is in combat, they are doing something wrong or things are already going very sideways.

**Comment by Winifridal** [comment]
Oh wait I thought the idea of this is that they would get teleported to their group and not to PLT, instead of not at all. So like when I crashed last session, I would get teleported to you (the group lead) and not to PLT.
